### PR TITLE
Network module configuration

### PR DESCRIPTION
**Pull request by AsysGuptaSF**
Screenshot from 2024-05-09 13-06-33
Screenshot from 2024-05-09 13-06-24
Screenshot from 2024-05-09 13-06-06
Screenshot from 2024-05-09 13-05-08
Screenshot from 2024-05-09 13-03-39

### CODE REVIEW

File: modules/network/README.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+## Outputs
+
+| Name | Description |
+|------|-------------|
+| <a name="output_eip_ids"></a> [eip\_ids](#output\_eip\_ids) | The IDs of the created Elastic IPs for NAT gateways |
+| <a name="output_nat_gateway_ids"></a> [nat\_gateway\_ids](#output\_nat\_gateway\_ids) | The IDs of the created NAT gateways |
+| <a name="output_private_subnet_ids"></a> [private\_subnet\_ids](#output\_private\_subnet\_ids) | The IDs of the created private subnets |
+| <a name="output_public_subnet_ids"></a> [public\_subnet\_ids](#output\_public\_subnet\_ids) | The IDs of the created public subnets |
+| <a name="output_vpc_id"></a> [vpc\_id](#output\_vpc\_id) | The ID of the created VPC |
+<!-- END_TF_DOCS -->
```
Comment (mayank0202) [inline modules/network/README.md]: `<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->`

Comment (AsysGuptaSF) [reply modules/network/README.md]: fixed

**Review by kumarabhinav19 — APPROVED** [review]
LGTM

**Review by mayank0202 — APPROVED** [review]
LGTM

**Pull request merged**